static/css/components/buttonCta.less

```
@@ -133,6 +133,16 @@ a.cta-btn {
   }
 }
 
+// Button inside carousel
+.carousel-section .cta-btn .btn-label {
+  padding: 8px 10px;
+  padding-right: 25px;
+  white-space: nowrap;
+  text-overflow: ellipsis;
+  font-size: 14px;
+  text-align: center;
+}
```

This should (1) make the 14px font-size apply to _all_ cta-btn's in carousels (some of them don't have a `btn-label`), and (2) add the right-padding only to buttons with the external icon, to prevent unnecessary wrapping.

```suggestion
.carousel-section .cta-btn {
  white-space: nowrap;
  text-overflow: ellipsis;
  font-size: 14px;
  text-align: center;

  &.cta-btn--external {
    padding-right: 25px;
    .btn-label {
       padding: 8px 10px;
    }
  }
}
```